tests/view/StatesTests.cpp

```diff
@@ -497,16 +498,42 @@ TEST_F(StatesTests, HelpStateExecute_ShouldPrintHelpMessage) {
 TEST_F(StatesTests, InputShowParametersStateExecute_ShouldCreateShowCommand) {
     // Arrange
     std::shared_ptr expected_next_state = std::make_shared<EndState>(nullptr);
+    std::shared_ptr<State> expected_state_machine_initial_state = std::make_shared<InputShowByLabelState>(nullptr);
+    std::unique_ptr<MockStateMachine> state_machine = std::make_unique<MockStateMachine>();
```

Comment:
You use `std::make_unique` here, but in `TaskManagerTest:576` you allocate with `new`. Be consistent and prefer `std::make_unique`.

Reply:
fixed